source/MaterialXCore/Util.cpp

```diff
@@ -62,6 +62,14 @@ bool isValidNamespace(const string& name)
     return isValidName(name) && name.find(':') == std::string::npos;
 }
 
+bool isValidEnvironmentPath(const string& path)
+{
+    // The column and semicolumn chars are used to separate paths in env vars on different os.
+    // Do not allow both of them to have more portable values.
+    return path.find(':') == std::string::npos 
```

Comment:
Shouldn't this be a different check based upon the OS?

Reply:
I wanted to have checked against both separators to make the pref variable file more portable between different OS.